app/test_main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+from unittest import mock
+
+import pytest
+
+from .main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "url,status,internet,result",
+    [
+        ("google.com", True, False, "Not accessible"),
+        ("gooogle.com", False, True, "Not accessible"),
+        ("google.com", True, True, "Accessible")
```

Comment:
no need to repeat url for 3 times, u can create this variable inside ur test only once

Reply:
also what about false/false condition ?